test/unit/Hooks.test.js

```diff
@@ -1,7 +1,14 @@
-import { marked } from '../../src/marked.js';
+import { Marked } from '../../lib/marked.esm.js';
 import { timeout } from './utils.js';
+import { describe, it, beforeEach } from 'node:test';
+import assert from 'node:assert';
```

Comment:
Its a matter of preference, but you could change this to `node:assert/strict` here and then `assert.strictEqual` becomes `assert.equal`. Or to make it more terse:

```
import { equal } from 'node:assert/strict';
```

Reply:
There are tests that can't use strict. I would rather keep it consistent.